odl/operator/pspace_ops.py

```diff
@@ -776,7 +852,7 @@ def prod_op(self):
 
     @property
     def operators(self):
-        """A tuple of sub-operators."""
+        """`tuple` of sub-operators."""
```

Comment:
On the other hand, since we try to adhere to the principle "be generous in what you accept and strict in what you return", there should be no doubt about the return type if an English word is used that has a corresponding Python type of the same name. 
We should assume that users know what a tuple is, and if we write `Tuple of foo`, what should the return type reasonably be apart from `tuple`? 
Again, with the same line of reasoning, we could easily start flagging all kinds of statements that "could be clearer".